WordPress/create-block-theme
# Use url attr to get the url of the media used by cover block

## What?
- Updates how the linked media gets from the `core/cover` block.
- Add a test case for cover blocks using background images.

## Why?
Fixes #748 

## How?
 It no longer parses the block's HTML but relies on its' URL' attribute.
 
 ## Testing instructions:
 - Add a cover block in a template using the site editor with an image from the media library.
 - Enable the 'Fixed background' or 'Repeated background' option in the cover block styles sidebar.
 - Save the changes in the editor.
 - Save the theme using CBT with the 'Localize Images' option enabled.
 - Check the code of core convert in the resulting pattern. It should reference an image from the theme with a relative URL, not the media library URL.
